ReactNativeApp/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ReactNativeApp",
```

Comment:
Whatever name we choose for the app (trustly-react-native-example) should be here too